### ISSUE TITLE
Do exit from popupmenu mode does not close the app

### ISSUE THREAD

**Issue by AkazaRenn** [HavenDV/H.NotifyIcon#66]
### Describe the bug

I know it sound ridiculous but really happened. I was testing the windowless sample [here](https://github.com/HavenDV/H.NotifyIcon/tree/master/src/apps/H.NotifyIcon.Apps.WinUI.Windowless) and realized the bug. Basically you just need to change `ContextMenuMode="SecondWindow"` in [the file](https://github.com/HavenDV/H.NotifyIcon/blob/master/src/apps/H.NotifyIcon.Apps.WinUI.Windowless/TrayIconResources.xaml#L112) to `ContextMenuMode="PopupMenu"` and try exiting the app from the context menu, you'll see the tray icon is go, but the process still stays in Task Manager, and Visual Studio does not quit debug mode either.

### Steps to reproduce the bug

1. Get the windowless sample from [here](https://github.com/HavenDV/H.NotifyIcon/tree/master/src/apps/H.NotifyIcon.Apps.WinUI.Windowless)
2. Change `ContextMenuMode="SecondWindow"` in [the file](https://github.com/HavenDV/H.NotifyIcon/blob/master/src/apps/H.NotifyIcon.Apps.WinUI.Windowless/TrayIconResources.xaml#L112) to `ContextMenuMode="PopupMenu"`
3. Build the app using `H.NotifyIcon` and `H.NotifyIcon.WinUI` from `NuGet`
4. Run the app in debug mode in Visual Studio
5. From tray icon, right-click to open the context menu, and hit `Exit`
6. Observe that Visual Studio is still in debug mode, and the process can still be found in the `Details` tab in Task Manager

### Expected behavior

The app should exit just like when it's using `SecondWindow`

### Screenshots

![Screenshot_20230118_073817](https://user-images.githubusercontent.com/8340896/213328607-384fc09c-c2f4-491e-96d6-525fc8c11e5b.png)

### NuGet package version

2.0.77

### Platform

WinUI

### IDE

Visual Studio 2022

### Windows Version

Windows 11

### WindowsAppSDK Version

Other

### WindowsAppSDK Type

Packaged

**Comment by myfix16** [comment]
I think it's related to the exit command. You can debug the `ExitApplicationCommand_ExecuteRequested` in `App.xaml.cs` and see if there is any problem.

**Comment by myfix16** [comment]
@AkazaRenn Yes, I can also replicate this bug. Before it's fixed, I suggest adding `System.Diagnostics.Process.GetCurrentProcess().Kill();` to `ExitApplicationCommand_ExecuteRequested` to terminate the app.
